rindexer_rust_playground/src/rindexer_lib/typings/rindexer_playground/events/playground_types_filter.rs

```diff
@@ -133,10 +131,10 @@ where
         Fut: Future<Output = EventCallbackResult<()>> + Send + 'static,
     {
         let csv = AsyncCsvAppender::new(
-            r"/Users/joshstevens/code/rindexer/rindexer_rust_playground/generated_csv/PlaygroundTypesFilter/playgroundtypesfilter-swap.csv",
+            r"/Users/jackedgson/Development/avara/rindexer/rindexer_rust_playground/generated_csv/PlaygroundTypesFilter/playgroundtypesfilter-swap.csv",
```

Comment:
we probs should also fix this later on to not have full paths like this

Reply:
Yeah, I'll rip my name out as well, would prefer it not to be in there